JitsiConferenceEventManager.js

```diff
@@ -586,6 +613,15 @@ JitsiConferenceEventManager.prototype.setupXMPPListeners = function() {
         });
 };
 
+/**
+ * Add XMPP listener and save its reference for remove on leave conference.
+ */
+JitsiConferenceEventManager.prototype._addConferenceXMPPListener = function(
+        eventName, listener) {
+    this.xmppListeners[eventName] = listener;
+    this.conference.xmpp.addListener(eventName, listener);
```

Comment:
Have you considered or looked into leveraging eventEmitter's removeAllListeners instead of maintaining an object of listeners? Not sure how safe that is though. The "Listenable" class is essentially a wrapped EventEmitter and adding a hook into removeAllListeners seems desirable. I'm sure there are a bunch more leaks in lib-jitsi-meet, as jitsi-meet relies on page reload after a meeting ends, and maybe having that convenience method would be nice, but I haven't looked into the edge cases.

Reply:
I was thinking to use removeAllListeners but I have to remove listeners by conference not by event type. I plan to join multiple rooms/conferences same time (only one with audio/video streams other rooms just for sending messages) and removeAllListeners would remove all conferences listener.

Reply:
I see. So each conference initializes its own JitsiConferenceEventManager and its own eventEmitter, but listeners get set on the xmppConnection itself, and the xmppConnection gets shared by different conferences.